scripts/monitor/rano_monitor/utils.py

```diff
@@ -387,3 +387,58 @@ def unpackage_reviews(file, app, dset_data_path):
             if os.path.exists(target_file):
                 delete(target_file, dset_data_path)
             tar.extract(member, dest)
+
+
+def brain_has_been_reviewed(brainpath, backup_brainpath):
+    if not os.path.exists(backup_brainpath):
+        return False
+
+    brain_hash = get_hash(brainpath)
+    backup_hash = get_hash(backup_brainpath)
+    return brain_hash != backup_hash
```

Comment:
To tell truth I don't understand the whole pipeline logics quite well. Can you explain plz why difference between brain_hash and backup_hash means anything was reviewed? For me it means only that the content was modified comparing to the backup...

Reply:
The brain mask will only change if someone opened the file and changed the contents, so this is why this check works to identify if the brain mask has been reviewed. We don't care about brain masks that have been opened but not modified.

As for tumor segmentations, there are situations in which someone might review the file but leave it unchanged, and we do need to identify those cases to be able to move forward. That's why for tumor segmentations we instead check the existence of that segmentation in a "finalized" folder. That file will only be there if a user puts it there (either manually or by using the monitoring tool "set as finalized" button).